package.json

```diff
@@ -28,7 +29,7 @@
     "publish-prerelease": "yarn lerna publish --preid development --skip-git --yes --canary",
```

Comment:
while we're at it, let's fix the `publish-npm`, `publish-manual-prerelease`, and `publish-prerelease` scripts to look exactly like the `next-release` command and the publish command.


https://github.com/RequestNetwork/requestNetwork/blob/683efd7f75a0b8decb7001c58ed9f592e7990da2/.circleci/config.yml#L268-L282
https://github.com/RequestNetwork/release-tools/blob/90942431172539f5240062389550c701e735257d/npm-publish.sh#L38
```sh
yarn lerna publish --conventional-commits --exact --message "chore(release): publish %s" --no-changelog --yes
```

Reply:
Changed!